tests/utils/command-utils.go

```diff
@@ -1,38 +1,26 @@
 package utils
 
 import (
-    "bytes"
-    "fmt"
-    "os"
-    "os/exec"
-    "path"
-    "runtime"
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"runtime"
 )
 
 func RunScriptFromBaseDir(command string, args []string, envVars []string) (string, string, error) {
-    _, filename, _, _ := runtime.Caller(0)
-    dir := path.Join(path.Dir(filename), "..", "..")
-    return RunCommand(fmt.Sprintf("%s/%s", dir, command), args, envVars)
+	_, filename, _, _ := runtime.Caller(0)
+	dir := path.Join(path.Dir(filename), "..", "..")
+	return RunCommand(fmt.Sprintf("%s/%s", dir, command), args, envVars)
 }
 
 func RunCommand(command string, args []string, envVars []string) (string, string, error) {
-    cmd := exec.Command(command, args...)
-    cmd.Env = append(os.Environ(), envVars...)
-    var stdout, stderr bytes.Buffer
-    cmd.Stdout = &stdout
-    cmd.Stderr = &stderr
-    err := cmd.Run()
-    return string(stdout.Bytes()), string(stderr.Bytes()), err
+	cmd := exec.Command(command, args...)
+	cmd.Env = append(os.Environ(), envVars...)
+	var stdout, stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err := cmd.Run()
+	return string(stdout.Bytes()), string(stderr.Bytes()), err
```

Comment:
Those changes are only due to `go fmt` running automatically on my machine - make sure to run that :)